exotations/exotica_core_task_maps/exotica_plugins.xml

```diff
@@ -62,4 +62,7 @@
   <class name="exotica/Manipulability" type="exotica::Manipulability" base_class_type="exotica::TaskMap">
     <description>Maximizes the manipulability measure using Yoshikawa's measure based on the velocity ellipsoid. Note, in-order to maximize manipulability the associated Rho must be negative. Derivatives are computed using finite differencing.</description>
   </class>
-</library>
\ No newline at end of file
+  <class name="exotica/AvoidLookAt" type="exotica::AvoidLookAt" base_class_type="exotica::TaskMap">
+    <description>Avoids looking at given target points. This can only be used as an inequality constraint.</description>
```

Comment:
Is there a way to make this a cost, e.g., using an epsilon?

Reply:
Using as cost is possible by modifying as potential field. Will add this in next commit.